src/modules/dashboard/lists/ListItemWrapper.tsx

```diff
@@ -1,6 +1,7 @@
 import { Tooltip, Typography } from '@mui/material';
 import { ReactNode } from 'react';
 import { BorrowDisabledToolTip } from 'src/components/infoTooltips/BorrowDisabledToolTip';
```

Comment:
Also (for now I don't know why), but the tooltip is display next to the token name instead of next to the APYs
(See on this USDS example)

![Capture d’écran 2025-02-13 à 16 00 01](https://github.com/user-attachments/assets/e12f9d86-dd79-4419-bdda-1a40ca1b3f0d)